src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -1845,9 +1845,28 @@ private void DoRunspaceInitialization(bool skipProfiles, string initialCommand,
                     sw.Stop();
 
                     var profileLoadTimeInMs = sw.ElapsedMilliseconds;
-                    if (profileLoadTimeInMs > 500 && s_cpp.ShowBanner)
+
+                    // Only check profile load time against threshold if we're actually going to show the message
+                    if (s_cpp.ShowBanner)
                     {
-                        Console.Error.WriteLine(ConsoleHostStrings.SlowProfileLoadingMessage, profileLoadTimeInMs);
+                        int showMessageThresholdMs = 500;
+
+                        int thresholdMs = PowerShellConfig.Instance.GetSlowProfileLoadingMessageThreshold(ConfigScope.AllUsers);
+                        if (thresholdMs > 0)
```

Comment:
What if the config contains 0 or -1 value?

Reply:
The current impl would ignore `<= 0` so you'd get the default 500 milliseconds.  I suppose  I could change that to `threshold >= 0` so the user to set it to 0 such that you'd always see the **slow** profile loading message.  That's a bit odd because this it is no longer about slow profile loading but is instead "always show my profile load time".  Then again a threshold of `1 ms` is more or less the same.  

Part of the problem with this approach to current user settings overriding the global setting is that I'm relying on a default value of 0 to indicate the setting is not set in that config file.  I should probably just make the PowerShellConfig property return `int?` and use that instead to see if the value was set in the corresponding config file.  Then 0 could literally mean the threshold is 0 ms and any negative number could mean the threshold is infinite.